extensions/vscode/webviews/homeView/src/components/TextStringWithAnchor.vue

```diff
@@ -0,0 +1,63 @@
+<template>
+  <span v-if="activeOption" :class="class">
+    {{ messageParts[0] }}
+    <a class="webview-link" role="button" @click="onClick()">
+      {{ activeOption?.anchorStr }}
+    </a>
+    {{ messageParts[1] }}
+  </span>
+  <span v-else :class="class">
+    {{ message }}
+  </span>
+</template>
+
+<script setup lang="ts">
+import { computed } from "vue";
+
+defineOptions({
+  inheritAttrs: false,
+});
+
+const emit = defineEmits<{
+  // <eventName>: <expected arguments>
+  click: [splitOptionId: number]; // named tuple syntax
+}>();
+
+interface SplitOption {
```

Comment:
May want to export this since it is used as prop, will make it easier to construct with type safety in other components.

Reply:
That has now been refactored into a common utility (and exported), since my next to last commit (counting this one which I'm making your requested changes on) needed the functionality within the log view.